apm-commons/apm-util/src/main/java/org/apache/skywalking/apm/util/StringUtil.java

```diff
@@ -62,4 +64,11 @@ public static boolean substringMatch(CharSequence str, int index, CharSequence s
         }
         return true;
     }
+
+    public static String cut(String str, int threshold) {
+        if (isEmpty(str) || str.getBytes().length <= threshold) {
+            return str;
+        }
+        return new String(Arrays.copyOf(str.getBytes(), threshold));
```

Comment:
Extract `str.getBytes()` to variable?

Reply:
ok，I will modify it later.

Reply:
What is the difference between this and `new String(value, beginIndex, subLen)` in `String#substring`?